.travis.yml

```diff
@@ -9,9 +9,6 @@ before_install:
   - "echo 'gem: --no-document' > ~/.gemrc"
   - git config --global user.name "Travis CI"
   - git config --global user.email "travis-ci@example.com"
-env:
```

Comment:
Can this be done in a separate PR?

Reply:
No problem. But I guess this change is temporarily needed because I tried to follow travis's documentation and looks like they are having some kind of bug right now.

Reply:
@teoljungberg https://github.com/teoljungberg/fx/pull/41